paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -209,15 +203,15 @@ internal fun PaymentMethodUI(
                 iconRes = iconRes,
                 iconUrl = iconUrl,
                 imageLoader = imageLoader,
-                color = color,
+                color = MaterialTheme.stripeColors.onComponent,
                 tintOnSelected = tintOnSelected
```

Comment:
do we also need to update the PaymentMethodIconUi? It still takes in a `tintOnSelected` param - not sure what that's used for/if we need to update

Reply:
The tint is used to ensure we have the correct color icon between light/dark mode.